cypress/e2e/fullstack-test.cy.js

```diff
@@ -78,16 +78,20 @@ describe('When I load the query tool', () => {
 describe('When I run an unfiltered query on all nodes', () => {
 
     beforeEach(() => {
-        cy.visit('http://localhost:3000/')
+        // We need to include the ?node=All otherwise it seems the app switches URLs at some point
+        // from http://localhost:3000/ to http://localhost:3000/?node=All,
+        // which is registered as a modification to query fields, thus preventing the results download
```

Comment:
Oh wow, great catch. I don't think we do that in any of the tests. @rmanaem take a look, we might want to look for this!